ThirdpartyToolchain.cmake

```diff
@@ -260,89 +259,142 @@ macro(build_icu4c)
                       --libdir=${ICU_LIBRARIES} ${ICU_CFG}
     BUILD_COMMAND ${MAKE_PROGRAM} -j ${NUM_JOBS}
     INSTALL_COMMAND ${HOST_ENV_CMAKE} ${MAKE_PROGRAM} install)
+
+  # We have to keep the FindICU.cmake in a subfolder to prevent it from
+  # overriding the system provided one when ICU_SOURCE=SYSTEM
+  list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/CMake/icu)
 endmacro()
 
 # ================================ END ICU4C ================================
 
-macro(build_dependency DEPENDENCY_NAME)
-  if("${DEPENDENCY_NAME}" STREQUAL "folly")
+# ================================== BOOST ==================================
+if(DEFINED ENV{VELOX_BOOST_URL})
+  set(BOOST_SOURCE_URL "$ENV{VELOX_BOOST_URL}")
+else()
+  # We need to sue boost > 1.70 to build it with CMake
```

Comment:
nit: s/sue/use

Reply:
also maybe change comment to say 1.80

Reply:
The boost repo contains cmake files starting in 1.70 so bit is correct though I have not tested it with a lower version and there were likely changes since then...